docs/reference/glossary.asciidoc

```diff
@@ -93,10 +111,16 @@ hence it is called a filter. Filters are simple checks for set inclusion or excl
 In most cases, the goal of filtering is to reduce the number of documents that have to be examined.
 // end::filter-def[]
 
+[[glossary-flush]] flush ::
+// tag::flush-def[]
+Make sure that all data in the transaction log is permanently written to the Lucene index. 
+{es} automatically triggers flushes as needed,
```

Comment:
```suggestion
{es} automatically triggers flushes as needed;.
```

Reply:
I think it's worth mentioning when calling flush is useful. (Preparing for upgrades.)